eslzArm/subscriptionTemplates/nvahubspoke-connectivity.json

```
@@ -235,6 +254,7 @@
                                                 "[concat('Microsoft.Network/virtualNetworks/', variables('hubName'))]"
                                             ],
                                             "properties": {
+                                                "activeActive": false,
```

```suggestion
                                                "activeActive": [if(equals(parameters('enableVpnActiveActive'), 'Yes'), 'true', 'false')],
```
You could try simplify the code using the if evaluation using the parameter.

---

I agree for setting the active/active flag, and that is how I originally had it.  Unfortunately, I cannot (to the best of my knowledge) use an if statement to add the additional active/active ip to the "ipConfigurations" block.  I can't use "condition" as that statement is only available at the root level, and not in the "ipConfigurations" block. Once I realized I would need to duplicate the code to add the additional active/active ip I simplified the active/active flag to just true and false.    Please let me know if there is a better way to use conditional logic to add the additional active/active ip in the "ipConfigurations" block.

---

Discussed with @terrymandin, and because additional set of parameters are required, this is a valid approach.